### DISCUSSION TITLE
[Bug]: Eloquent model values not supported

### DISCUSSION THREAD

**Discussion by simon-tma** [spatie/tabular-assertions#6] [Q&A]
### What happened?

Attempting to match against an array of Eloquent models throws an error

### How to reproduce the bug

```php
use App\Models\User;

$users = [
    new User(['id' => 20, 'name' => 'John Doe', 'email' => 'john@doe.com']),
    new User(['id' => 1245, 'name' => 'Jane Doe', 'email' => 'jane@doe.com']),
];

expect($users)->toMatchTable('
    |   id | name        | email           |
    |   20 | John Doe    | john@doe.com    |
    | 1245 | Jane Doe    | jane@doe.com    |
');
```

### Package Version

1.0.0

### PHP Version

8.3.0

### Laravel Version

10.42.0

### Notes

Workaround is to call `getAttributes()` on each model before comparing

**Comment by sebastiandedeyne** [comment]
Not really a bug, the package is currently built to deal with arrays. I wouldn't mind using a different accessing method to have it more flexible for objects.
